CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Styles for Contributing to Documentation
+
+
+Keeping consistent styles enables Sphinx to be able to create a proper
+hierarchy of headings in output files such as PDFs and HTML. Use the styles
+below when editing your ReST files.
+
+.. code-block::
+
+   ##############
+   Document title
+   ##############
+
+   Heading 1
+   ---------
+
+   Heading 2
+   *********
+
+   Heading 3
+   =========
+
+   Heading 4
+   ^^^^^^^^^
```

Comment:
This block is automatically formatted in Markdown and will look confusing if one reads .md file not in raw mode, insert it as Markdown code block using:

\`\`\`
[RST formatting examples]
\`\`\`